static/js/actions/email_test.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import { assert } from 'chai';
+
+import {
+  actionCreatorGenerator,
+  startEmailEdit,
+  updateEmailEdit,
+  clearEmailEdit,
+
+  START_EMAIL_EDIT,
+  UPDATE_EMAIL_EDIT,
+  CLEAR_EMAIL_EDIT,
+} from './email';
+
+describe('email actions', () => {
+  describe('actionCreatorGenerator', () => {
+    it('should return a function', () => {
+      assert.isFunction(actionCreatorGenerator(START_EMAIL_EDIT));
+    });
+
+    it('returned function should be a simple action creator with no args', () => {
```

Comment:
some possibly-clearer alternatives to this description and the one on line 26

'should return a simple action creator function when no args are provided'

'should return an action creator function with payload when args are provided'